### PR TITLE
Created Endpoints for Mentor and Mentee Read

### PR DESCRIPTION
**Pull request by AdamHVu**
## Description

Created Endpoints for Mentors and Mentees to check their data, in this case using a "read" endpoint, given specific queries.
This was made in conjunction with Create and Update so that users could submit their information, check if it needs updates, and of course update it accordingly. I also fixed a minor bug regarding function names so that has been incorporated.

Fixes # (issue)

## Type of change
- [X] New feature

## Checklist:

- [X] My code follows PEP8 style guide
- [X] I have removed unnecessary print statements from my code
- [X] I have made corresponding changes to the documentation if necessary
- [X] My changes generate no errors
- [X] No commented-out code
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

My Loom Video: https://www.loom.com/share/4129501ebea2429e8495c0cb7764c56f

### CODE REVIEW

**Review by mikayla-kosmala — APPROVED** [review]
small change?

**Review by VABIII — APPROVED** [review]

**Comment by AdamHVu** [comment]
I am reopening this PR because I was asked to due to conflicts. The change that is shown here is not a change that needs to be kept as I can revert it but the main gist of this PR is to add the read endpoints. I have as well fixed a bug regarding function names and have added that change as well.

File: app/api.py [hunk 1]
```diff
@@ -58,46 +58,35 @@ async def collections():
...
 async def computer_assignment_rating_visualizer():
     """Return an HTML table of the computer assignment
-    ratings in the computer assignment collection of the 
+    ratings in the computer assignment collection of the
     selected mongodb database.
     """
     return computer_assignment_visualizer(API.db)
 
 
-@API.post("/update/mentors")
```
Comment (BrokenShell) [inline app/api.py:67]: Why is this being deleted?

**Review by BrokenShell — CHANGES_REQUESTED** [review]

File: app/api.py (hunk 1 above)
Comment (AdamHVu) [reply app/api.py:67]: I have fixed the missing update and delete endpoints! Refactoring will fix the final error.

Comment (BrokenShell) [reply app/api.py:67]: You have some PEP8 issues. Do you use a linter or IDE with automatic linting?

Comment (AdamHVu) [reply app/api.py:67]: As of now, I use Flake8 which I did enable and cleaned up any errors and whitespaces I had missed previously

**Review by BrokenShell — APPROVED** [review]

**Review by mikayla-kosmala — APPROVED** [review]
looking good

**Pull request merged**